rundeckapp/grails-app/controllers/rundeck/controllers/FrameworkController.groovy

```diff
@@ -3523,5 +3523,31 @@ class FrameworkController extends ControllerBase implements ApplicationContextAw
         }
         render(status: HttpServletResponse.SC_NO_CONTENT)
     }
+
+    def evaluateKeyStorageAccessNodeResource(){
+        withForm {
+            g.refreshFormTokensHeader()
+
+            if (!params.project) {
+                return renderErrorView("Project parameter is required")
+            }
+
+            String keyPath = "keys/"
```

Comment:
I don't think the path can be hardcoded to check access, since ACL might be defined for a subpath anyway. I think the ajax check for access should probably be removed

Reply:
I will remove this one, I will check the resource model errors like healthcheck